mplc/partner.py

```diff
@@ -41,3 +41,50 @@ def corrupt_labels(self):
     def shuffle_labels(self):
         for label in self.y_train:
             label = shuffle(label)
+
+
+class PartnerMpl:
+
+    def __init__(self, partner, mpl):
+        """
+        :type partner: Partner
+        :type mpl: MultiPartnerLearning
+        """
+        self.mpl = mpl
+        self.id = partner.id
+        self.batch_size = partner.batch_size
+        self.minibatch_count = mpl.minibatch_count
+        self.partner_parent = partner
+        self.model_weight = None
+
+        self.minibatched_x_train = np.nan * np.zeros((self.minibatch_count, self.batch_size))
+        self.minibatched_y_train = np.nan * np.zeros((self.minibatch_count, self.batch_size))
```

Comment:
I don't get why there is a second dimension of size `self.batch_size`

Reply:
I missunderstood the role of self.batch_size. I remove this second dimension